Makefile

```diff
@@ -80,6 +81,9 @@ ifeq ($(SOURCE_OS),darwin)
 	SUBST = $(HOMEBREW_PREFIX)/Cellar/x264/r3108/lib/libx264.a
 endif
 CGO_LDFLAGS = $(subst -lx264, $(SUBST),$(shell PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) pkg-config --libs $(FFMPEG_LIBS))) 
+ifeq ($(TARGET_OS),windows)
+	CGO_LDFLAGS += -static -static-libgcc -static-libstdc++
```

Comment:
We need this to statically link libc and cpp standard library

Reply:
why do we need this for windows and not other oses?

Reply:
yep, why is this needed?